src/pydagmc/dagnav.py

```diff
@@ -479,6 +479,20 @@ def triangle_coords(self):
 
         return self.model.mb.get_coords(conn.flatten()).reshape(-1, 3)
 
+    @property
+    def bounds(self):
+        """Returns the axis-aligned bounding box for all triangles under this set.
+
+        Returns
+        -------
+        tuple(float, float, float, float, float, float)
+            (xmin, xmax, ymin, ymax, zmin, zmax)
+        """
+        coords = self.triangle_coords
+        min_coord = np.min(coords, axis=0)
+        max_coord = np.max(coords, axis=0)
+        return min_coord, max_coord
```

Comment:
Why isn't this `(xmin, ymin, zmin), (xmax, ymax, zmax)`?  Is it really a tuple? or is it a pair of numpy arrays that can be iterated like a tuple like they are in the tests?